app/components/shared/qualifications_table_component.html.erb

```diff
@@ -2,6 +2,7 @@
 <% if qualifications.any? %>
   <h5 class="govuk-heading-s govuk-!-margin-top-6" id="other-qualifications"><%= subheader %></h5>
   <table class="govuk-table govuk-!-width-full" data-qa="qualifications-table-<%= header.parameterize %>">
+    <caption id="other-qualifications" class="govuk-visually-hidden">Details of A levels and other qualifications</caption>
```

Comment:
We can just move the h5 into the caption. In other words, delete line 3 and change this:

```suggestion
    <caption id="other-qualifications">
    <h5 class="govuk-heading-s govuk-!-margin-top-6" id="other-qualifications"><%= subheader %>
    </h5>
    </caption>
```

The way it is implemented above, a screen reader will read the heading and then the caption, which presumably says the same thing.